types/react-native/index.d.ts

```diff
@@ -162,6 +162,31 @@ interface EmitterSubscription extends EventSubscription {
     remove(): void;
 }
 
+interface DEPRECATED_RCTExport<T> {
+	getConstants(): Function;
+}
+
+interface TurboModule extends DEPRECATED_RCTExport<void> {}
+
+interface Spec extends TurboModule {
+    addListener(eventName: string): void;
+    removeListeners(count: number): void;
+}
+
+declare class TurboModuleRegistry {
```

Comment:
`TurboModuleRegistry` isn’t a class, but rather a module. Where is the export you’re using? Or are you importing from `Libraries/Components/AppleTV/NativeTVNavigationEventEmitter`?

Reply:
yes, it's a module & I'm trying to import it from the Libraries/Components/AppleTV/NativeTVNavigationEventEmitter

Reply:
Right, so we can’t have that in this root file, it needs to be in a file that follows the same package dir structure, otherwise this won’t work. Take a look at `relay-runtime` for an example of this.

Reply:
Hi @alloy,
you meant that it has to be split out from index.d.ts / root file and keep it inside a lib folder?

Reply:
Correct, otherwise the types won’t be picked up by TypeScript when you import from that file directly. This is also why I asked for test coverage, because you will see that if you add test real-world coverage, it should fail as-is.